pom.xml

```
@@ -5,7 +5,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.epam.prejap</groupId>
     <artifactId>tetris</artifactId>
-    <version>0.10</version>
+    <version>1.1</version>
```

```suggestion
    <version>0.11</version>
```
It's not a general rule that the the minor version number overflows at 10 and causes increase of the major number. Some projects do that, but many don't, and it's rather an explicit decision of the team.